LayoutTests/inspector/css/getMatchedStylesForNodeNestingStyleGrouping.html

```diff
@@ -112,6 +147,16 @@
     #outerA {
         color: red;
 
+        @media(min-width: 0px) {
+            background-color: blue;
```

Comment:
is this actually useful/needed?

Reply:
This line mostly exists so that the test exercises the potentially less common path of the property we want being below an explicitly nested rule.